pages/portfolio.js

```diff
@@ -0,0 +1,155 @@
+/* global BACKEND_URL, fetch */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Layout from '../components/layout/main';
+import Background from '../components/content/background';
+
+import Works from '../components/portfolio/works';
+
+import { works } from '../main.config';
+
+const transformateCategories = (choosenCategory, existCategories) => {
```

Comment:
@dmytropanontko , "chosen" should have 1 "o". As for "existCategories", let's call it "filterList" to keep it consistent with the var name below.